pkg/controller/studyjob/studyjob_controller.go

```diff
@@ -406,6 +408,53 @@ func (r *ReconcileStudyJobController) updateWorker(c katibapi.ManagerClient, ins
 	return update, nil
 }
 
+func(r *ReconcileStudyJobController) getJobWorkerStatus(w *katibv1alpha1.WorkerCondition, ns string) WorkerStatus {
```

Comment:
nit: s/func(/func (

@hougangliu are you using golint to check your code format?

Reply:
Ah, I see, thanks